modules/cnf-configuring-log-filtering-for-linuxptp.adoc

```diff
@@ -4,13 +4,12 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cnf-configuring-log-filtering-for-linuxptp_{context}"]
-= Configuring log filtering for linuxptp services
+= Configuring log filtering for PTP
 
-The `linuxptp` daemon generates logs that you can use for debugging purposes. In telco or other deployment types that feature a limited storage capacity, these logs can add to the storage demand.
-
-To reduce the number log messages, you can configure the `PtpConfig` custom resource (CR) to exclude log messages that report the `master offset` value. The `master offset` log message reports the difference between the current node's clock and the master clock in nanoseconds.
+Modify the `PtpConfig` custom resource (CR) to configure basic log filtering and exclude log messages that report the master offset value.  
```

Comment:
Could you use something else in place of master?

Reply:
no alternative